src/QMCWaveFunctions/BsplineFactory/SplineR2R.cpp

```diff
@@ -48,6 +48,113 @@ bool SplineR2R<ST>::write_splines(hdf_archive& h5f)
   return h5f.writeEntry(bigtable, o.str().c_str()); //"spline_0");
 }
 
+/* 
```

Comment:
Customarily the documentation goes in the header this is the impmlementation and documentation in this file is expected to relate to that.

Reply:
I moved some documentation the header file. But I wanted to leave the comments in the function body because I'm going to generalize to other spline classes and will likely need some breadcrumbs.